src/cmd/service.go

```diff
@@ -228,7 +229,7 @@ var deleteServiceTagCmd = &cobra.Command{
 			cobra.CheckErr(fmt.Errorf("service '%s' not found", serviceKey))
 		}
 		for _, tag := range result.Tags.Nodes {
-			if tagKey == tag.Key {
+			if tagKey == tag.Key || tagKey == tag.Id {
```

Comment:
This works; but causes unnecessary looping.  We have a function to determine if the passed in string is an ID so it would eliminate the need for looping over all the tags in the case of the user passing in an ID instead of a tag key name.

For example:
```go
		if common.IsID(tagKey) {
			err := getClientGQL().DeleteTag(tagKey)
			cobra.CheckErr(err)
			fmt.Println("Deleted Tag")
		} else {
			for _, tag := range result.Tags.Nodes {
				if tagKey == tag.Key {
```

I also threw in some error handling since it seems we were not doing that properly here.

Reply:
Thanks, I'll update my PR to reflect this change now.